src/index.js

```diff
@@ -12,6 +13,37 @@ import { bindActionCreators } from 'redux';
 import MainMenuScreen from './screens/MainMenuScreen';
 
 class HomeScreen extends React.Component {
+  componentDidMount() {
+    this.loadResourcesAsync();
+    Notifications.addNotificationReceivedListener(notification => {
+      console.log('notification ------------------', notification);
+    })
+
+    // Listen to notification interaction
+    this.notifTapSubscription = Notifications.addNotificationResponseReceivedListener(response => {
+      const data = response.notification.request.content.data;
```

Comment:
Are we sure this tree will always come or we need a proofer solution?

Reply:
This tree is working fine and also for backed we have to send the data for proposal and results in this tree format.